### PR TITLE
Feature/3100  async tlm publish

### PR DESCRIPTION
**Pull request by grignards**
Refs #3100.

## Checkpoints

The goal of this section is to help code reviewers and the merge request author to do the required checks on this pull request. Please don't edit this list.
If one or more of these checkpoints are out of scope, please write below why they are.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, then there are associated issues and issue is referenced in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present
- [x] Automated tests have been written (covering the feature or non regression), or *at least* an issue has been created to implement the test

**For developers**

- [x] Manual tests done to ensure stability on whole application layers, issue expectation follow
- [x] The original issue has been updated with a short summary of the implemented solution

**For the quality team**

- [x] Tested by the quality team

### CODE REVIEW

File: backend/cypress_test.ini [hunk 1]
```diff
@@ -209,7 +209,7 @@ email.template_dir = %(here)s/tracim_backend/templates/mail
 # processing_mode may be sync or async,
 # if async is choosen, you need also to run mail_notifier daemon
 # see README for more info.
-email.processing_mode = sync
+jobs.processing_mode = sync
 ## with async, please also configure redis below
 ; email.async.redis.host = localhost
```
Comment (inkhey) [inline backend/cypress_test.ini:214]: change these commented value too for consistency (NB: cypress_test.ini should probably be deleted or simplify later)

File: backend/tracim_backend/tests/utils.py [hunk 2]
```diff
@@ -245,6 +246,23 @@ def last_event(self) -> typing.Optional[Event]:
         return self._session.query(Event).order_by(Event.event_id.desc()).limit(1).one()
 
 
+class DockerCompose:
+    command = [
+        "docker-compose",
```
Comment (inkhey) [inline backend/tracim_backend/tests/utils.py:251]: Docker compose should be added as requirement somewhere in test documentation.
It should also be installed in travis (already done ?)

File: backend/development.ini.sample [hunk 3]
```diff
@@ -331,18 +331,23 @@ website.base_url = %(basic_setup.website_base_url)s
...
-### Email sending configuration ###
+### configuration of potentially lengthy jobs ###
 # processing_mode may be sync or async,
-# if async is choosen, you need also to run mail_notifier daemon
-# see README for more info.
-; email.processing_mode = sync
+# if async is choosen, you need also to run:
+#  - a custom rq worker (see backend/README.md)
+#  - a redis server
+; jobs.processing_mode = sync
```
Comment (inkhey) [inline backend/development.ini.sample:342]: This is a breaking change, should be notified in https://github.com/tracim/tracim/issues/2785.

Comment (grignards) [reply backend/development.ini.sample:342]: Done in https://github.com/tracim/tracim/issues/2785#issuecomment-637544988

File: backend/tracim_backend/tests/utils.py (hunk 2 above)
Comment (grignards) [reply backend/tracim_backend/tests/utils.py:251]: Enhanced the documentation for this, especially as I also use it for frontend dev/tests.

File: backend/cypress_test.ini (hunk 1 above)
Comment (grignards) [reply backend/cypress_test.ini]: Changed.

**Pull request merged**